src/styles/withTheme.js

```diff
@@ -0,0 +1,33 @@
+// @flow weak
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import wrapDisplayName from 'recompose/wrapDisplayName';
```

Comment:
Oh, good catch, then using  the `createEagerFactory` should be a better alternative to the `wrapDisplayName` helper.